docs/sdk/dev/development-tools/ts-linter.md

```diff
@@ -53,7 +53,7 @@ For details about how to install the TS Linter for your project, see the [TS Lin
 
 To execute the TS Linter, do the following:
```

Comment:
Please add a note at the beginning of the document that it has been migrated to ESLint since `202304.0`